### PR TITLE
Enhance `<Nav />` Component with `footerLabel` and `footerLogo` Props

### PR DESCRIPTION
**Pull request by cmarin001**
This pull request enhances the `<Nav />` component by introducing two new optional props: `footerLabel` and `footerLogo`. These props provide greater flexibility in customizing the footer content of the `<Nav />` component.

### CODE REVIEW

File: src/Nav/index.tsx [hunk 1]
```diff
@@ -219,17 +229,21 @@ const Nav = (props: INav) => {
...
+                appearance={navCopyrightAppearance}
+                padding="24px"
+                textAlign="start"
+                weight="bold"
+              >
+                {footerLabel}
+              </Text>
+            )}
           </Stack>
         </StyledFooter>
```
Comment (wfercanas) [inline src/Nav/index.tsx:248]: Adjust footer to control padding:
Screenshot from 2024-09-04 12-05-03

File: src/Nav/index.tsx [hunk 2]
```diff
@@ -219,17 +229,21 @@ const Nav = (props: INav) => {
...
-              appearance={navCopyrightAppearance}
-              padding="24px"
-              textAlign="start"
-              weight="bold"
-            >
-              {year} - Inube
-            </Text>
+          <Stack justifyContent="center" alignItems="center">
+            {footerLogo ? (
+              <StyledFooterLogoImage src={footerLogo} alt="Footer Logo" />
```
Comment (wfercanas) [inline src/Nav/index.tsx:234]: Let's leave the alt empty: `alt=""`. This is used when we don't want to give a meaning to an image, when it is just decorative.

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/Nav/index.tsx (hunk 2 above)
Comment (cmarin001) [reply src/Nav/index.tsx:234]: solved

File: src/Nav/index.tsx (hunk 1 above)
Comment (cmarin001) [reply src/Nav/index.tsx:248]: solved

**Pull request merged**